o1vm/src/pickles/proof.rs

```diff
@@ -23,3 +24,12 @@ impl<G: KimchiCurve> ProofInputs<G> {
         }
     }
 }
+
+// FIXME: should we blind the commitment?
+pub struct Proof<G: KimchiCurve> {
+    pub commitments: WitnessColumns<PolyComm<G>>,
```

Comment:
we'll need more explicit names when we'll have more polynomials than the witness columns, but ok for now